.github/workflows/cpp.yml

```
@@ -197,12 +197,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os:
-          - windows-2019
         include:
           - os: windows-2019
-            name: Windows 2019
+            name: Windows 2019 x64
             generator: Visual Studio 16 2019
+            platform: x64
+            openssl-n-bits: 64
+            cmake-args: -A x64 -DOPENSSL_ROOT_DIR="C:\Program Files\OpenSSL-Win64"
+            chocolate-force:
+          - os: windows-2019
+            name: Windows 2019 Win32
```

Win32 is the former name of the Windows API
```suggestion
            name: Windows 2019 x86
```